apis/validator/block.v3.yaml

```diff
@@ -59,10 +59,16 @@ get:
             title: ProduceBlockV3Response
             type: object
             properties:
-              version:
+              consensus-version:
```

Comment:
Unless there's a strong need to namespace another `.*version.*` property would prefer to keep the previous name for consistency

Reply:
we use 'version' in several endpoints in the metadata, should probably leave it at that I guess, but I'm not going to die on that hill...

Reply:
Okay put it back to just 'version'.